lib/puppet_litmus/rake_tasks.rb

```diff
@@ -83,6 +83,28 @@ def run_local_command(command)
     end
   end
 
+  desc "provision list of machines from provision.yaml file. 'bundle exec rake 'litmus:provision_list[default]'"
+  task :provision_list, [:key] do |_task, args|
+    provision_hash = YAML.load_file('./provision.yaml')
+    provisioner = provision_hash['default']['provisioner']
```

Comment:
Some nice error if `default` doesn't exist would be nice. Same below for the `key`.